plugin-build/src/test/kotlin/io/sentry/android/gradle/integration/SentryPluginSourceContextTest.kt

```diff
@@ -55,6 +58,92 @@ class SentryPluginSourceContextTest :
         assertTrue(result.output) { "BUILD SUCCESSFUL" in result.output }
     }
 
+    @Test
```

Comment:
`l` should we add more tests to check whether changing an input file actually reruns the task?

Reply:
This would only make sense for taks `SentryUploadProguardMappingsTask` and `UploadSourceBundleTask` which are, at the moment always executed, due to the `cliExecutable` changing at every run (see NextSteps of the PR description). Once that issue is resolved, it would absolutely make sense to create such a test, especially to make sure the behaviour hasn't changed in future gradle versions.